backend/tests/conftest.py

```diff
@@ -32,3 +32,18 @@ def vcr_config():
         ),
         "record_mode": "once",
     }
+
+
+@pytest.fixture(scope="module")
+def es(appctx):
+    """Setup and teardown all registered Elasticsearch indices.
+
+    Scope: module
```

Comment:
redundant :)

Reply:
This fixture is required to overwrite invenio es fixture which also deletes indexes.